TodoList/sources/TodoList.Repository/Dependency/RegisterTypes.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using Microsoft.Practices.Unity;
 using TodoList.Contracts.Dependency;
 using TodoList.Contracts.Repository;
+using TodoList.Repository.Helpers;
 
 namespace TodoList.Repository.Dependency
 {
     public class RegisterTypes : IBootstrapper
     {
         public IUnityContainer RegisterType(IUnityContainer container) 
-            => container.RegisterType<INodesRepository, NodesRepository>(new HierarchicalLifetimeManager());
+            => container.RegisterType<INodesRepository, NodesRepository>(new InjectionConstructor(ConnectionHelper.GetDbConnection()));
```

Comment:
You do not specify a lifetime manager here (making it [transient](https://msdn.microsoft.com/en-us/library/microsoft.practices.unity.transientlifetimemanager.aspx)) and you really should. Read through `Mongo` driver's documentation and select the correct manager, please.